cmd/benchmark.go

```diff
@@ -248,6 +251,11 @@ func systemCommandAction(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("locating package root failed: %w", err)
 	}
 
+	profile, err := cobraext.GetProfileFlag(cmd)
+	if err != nil {
+		return nil
```

Comment:
I have seen this now while updating https://github.com/elastic/elastic-package/pull/1230, shouldn't this be `return err`?

Reply:
That's right, it should return the error.
Thanks for the heads up! 

Opened a PR for this: https://github.com/elastic/elastic-package/pull/1370